### ISSUE TITLE
Graphical way to add apps to Shortcuts → Applications

### ISSUE THREAD

**Issue by moonmehta** [elementary/settings-keyboard#161]
In the "Applications" pane as seen below:
![screenshot from 2018-05-03 15-17-17](https://user-images.githubusercontent.com/22585732/39570389-d1e9c010-4ee5-11e8-9887-4e5898ca53d6.png)

Use a GUI similar to the "Startup applications" to be able to easily assign keyboard shortcuts to any app. 
![screenshot from 2018-05-03 15-18-32](https://user-images.githubusercontent.com/22585732/39570468-0f4cef90-4ee6-11e8-8eea-292b15c3a884.png)

This is better because its not easy to find terminal names for all the apps.

**Comment by uncertainquark** [comment]
This would also allow people to easily assign keyboard shortcuts to their favorite apps.

**Comment by cassidyjames** [comment]
This could also give us a nice place for app developers to point users to for adding shortcuts to launch Flatpak apps, e.g. `settings://input/keyboard/shortcuts/applications`

**Comment by teamcons** [comment]
While it looks different this one still seem actual.

Why not reuse the application selector already present in Systemsettings->Application->When logging in
(Second "System" item) ?
This already allows to either select an app, or type in magic commandline spookery.
It is consistent with Startup applications. It's even the same kind of button.

It isnt the same kind of list. Maybe the same design can be reused, with shortcuts and gear in place of toggle.

![Bildschirmfoto von 2024-10-16 19 55 57](https://github.com/user-attachments/assets/4cf4351e-4de5-4cfa-8864-25ec1e7700d9)

**Comment by danirabbit** [comment]
Yes re-using the design from Applications settings is a good move here, it just needs someone to do the implementation
